java-runtime/ftl-runtime/deployment/src/main/java/xyz/block/ftl/deployment/VerbClientsProcessor.java

```diff
@@ -0,0 +1,159 @@
+                            var publish = cc.getMethodCreator("call", returnType.name().toString());
+                            var helper = publish.invokeStaticMethod(MethodDescriptor.ofMethod(VerbClientHelper.class, "instance", VerbClientHelper.class));
+                            var results = publish.invokeVirtualMethod(MethodDescriptor.ofMethod(VerbClientHelper.class, "call", Object.class, String.class, String.class, Object.class, Class.class, boolean.class, boolean.class), helper, publish.load(name), publish.load(module), publish.loadNull(), publish.loadClass(returnType.name().toString()), publish.load(false), publish.load(false));
+                            publish.returnValue(results);
+
+                            publish = cc.getMethodCreator("call", Object.class);
+                            helper = publish.invokeStaticMethod(MethodDescriptor.ofMethod(VerbClientHelper.class, "instance", VerbClientHelper.class));
+                            results = publish.invokeVirtualMethod(MethodDescriptor.ofMethod(VerbClientHelper.class, "call", Object.class, String.class, String.class, Object.class, Class.class, boolean.class, boolean.class), helper, publish.load(name), publish.load(module), publish.loadNull(), publish.loadClass(returnType.name().toString()), publish.load(false), publish.load(false));
+                            publish.returnValue(results);
+                            clients.put(iface.name(), new VerbClientBuildItem.DiscoveredClients(name, module,cc.getClassName()));
+                        }
+                        found = true;
+                        break;
+                    } else {
+                        throw new RuntimeException("@VerbClientDefinition can only be applied to interfaces that directly extend a verb client type with concrete type parameters and " + iface.name() + " does not have concrete type parameters");
+                    }
+                } else if (i.name().equals(VERB_CLIENT_EMPTY)) {
+                        try (ClassCreator cc = new ClassCreator(classOutput, iface.name().toString() + "_fit_verbclient", null, Object.class.getName(), iface.name().toString())) {
+                            if (launchModeBuildItem.isTest()) {
+                                cc.addAnnotation(TEST_ANNOTATION);
+                                cc.addAnnotation(Singleton.class);
+                            }
+                            var publish = cc.getMethodCreator("call", void.class);
+                            var helper = publish.invokeStaticMethod(MethodDescriptor.ofMethod(VerbClientHelper.class, "instance", VerbClientHelper.class));
+                            publish.invokeVirtualMethod(MethodDescriptor.ofMethod(VerbClientHelper.class, "call", Object.class, String.class, String.class, Object.class, Class.class, boolean.class, boolean.class), helper, publish.load(name), publish.load(module), publish.loadNull(), publish.loadClass(Void.class), publish.load(false), publish.load(false));
```

Comment:
This is a long line! Would it be useful to split it up a bit?

Reply:
I need to setup auto formatting.